### PR TITLE
Several Text Changes

### LINKED ISSUE #153: [BUG] Several Text Changes
## Description

* Email template: This should not start with "Congratulations" to the admin. Idk why we are congratulating them lol.
* Onboarding slide: Slide 8: "Are you able to foster dogs that DO NOT do well with" -> this needs to be something positive/more straight forward for better UX
* "Use my Preferences" Button: "Use my Settings" or "Import from profile" or equivalent
* "Volunteer preferences" in the email template needs to say capabilities

### PR DESCRIPTION
**Pull request by uma-anand**
## Several Text Changes

Issue Number(s): #153 .

What does this PR change and why?
change wording of onboarding, preferences, and capabilities

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: web/components/Feed/Feed.tsx [hunk 1]
```diff
@@ -422,7 +422,7 @@ function Feed(props: {
           }}
           variant="solid-primary"
         >
-          Use My Preferences
+          Use My Settings
```
Comment (katsukixyz) [inline web/components/Feed/Feed.tsx]: Let's use "Fill from profile" instead

File: web/components/Feed/Feed.tsx [hunk 2]
```diff
@@ -498,7 +498,7 @@ function Feed(props: {
               }}
               variant="solid-primary"
             >
-              Use My Preferences
+              Use My Settings
```
Comment (katsukixyz) [inline web/components/Feed/Feed.tsx]: See above

File: web/email/email-template.ts [hunk 3]
```diff
@@ -276,7 +276,7 @@ export function populateEmailTemplate(
                 <table class="info-table">
                   <tr>
                     <th>Preferences & Info</th>
-                    <th>Volunteer Preferences</th>
+                    <th>Capabilities</th>
```
Comment (katsukixyz) [inline web/email/email-template.ts]: Change to "Volunteer Capabilities"

File: web/email/template.html [hunk 4]
```diff
@@ -213,7 +213,7 @@
         <table class="info-table">
           <tr>
             <th>Preferences & Info</th>
-            <th>Volunteer Preferences</th>
+            <th>Capabilities</th>
```
Comment (katsukixyz) [inline web/email/template.html]: See above

File: web/pages/onboarding.tsx [hunk 5]
```diff
@@ -272,7 +272,7 @@ function Onboarding() {
     } as StoredQuestion<Age>,
     {
       key: "dogsNotGoodWith",
-      title: "Are you able to foster dogs that DO NOT do well with:",
+      title: "Are you able to foster dogs that may not be comfortable with:",
```
Comment (katsukixyz) [inline web/pages/onboarding.tsx]: Let's revert this to before for full clarity to the user

**Review by katsukixyz — CHANGES_REQUESTED** [review]
Requesting some changes to the text - we should've discussed these as an LT before writing this bug. Sorry about the changes!

**Review by katsukixyz — APPROVED** [review]
